prac_05/state_names.py

```diff
@@ -5,14 +5,18 @@
 """
 
 # TODO: Reformat this file so the dictionary code follows PEP 8 convention
-CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
-                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
-print(CODE_TO_NAME)
+STATE_NAMES = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+               "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+print(STATE_NAMES)
 
-state_code = input("Enter short state: ")
-while state_code != "":
-    if state_code in CODE_TO_NAME:
-        print(state_code, "is", CODE_TO_NAME[state_code])
+state = input("Enter short state: ").upper()
+while state != "":
+    if state in STATE_NAMES:
+        print(state, "is", STATE_NAMES[state])
     else:
         print("Invalid short state")
-    state_code = input("Enter short state: ")
+    state = input("Enter short state: ").upper()
+
+# loop with formatting
+for state in STATE_NAMES:
+    print("{:3} is {}".format(state, STATE_NAMES[state]))
```

Comment:
You could use list comprehension to get the max length of the keys in the dictionary to then apply it to the .format() function for the short state code

Reply:
great thanks,